### PR TITLE
fixes ENS to new registry

### PR DESCRIPTION
**Pull request by decanus**
uses new ENS registry. see: https://github.com/ensdomains/ens/security/advisories/GHSA-8f9f-pc5v-9r5h

### CODE REVIEW

File: src/status_im/ethereum/ens.cljs
Comment (3esmit) [inline src/status_im/ethereum/ens.cljs]: ropsten address dont changes?

**Comment by corpetty** [comment]
any particular thing this is waiting on?

**Comment by decanus** [comment]
@corpetty code review and merge button press.

**Review by corpetty — APPROVED** [review]
lgtm, unless there is another place in the code referencing ENS that was missed.

**Comment by decanus** [comment]
I couldn't find anything else.

**Review by yenda — APPROVED** [review]

**Comment by yenda** [comment]
@Serhy please make a quick test of ENS relates features

@decanus lgtm, ideally we should move the contract address out of the translations but it can be done later

**Review by 3esmit — CHANGES_REQUESTED** [review]
testnet address dont changes?

File: src/status_im/ethereum/ens.cljs
Comment (decanus) [reply src/status_im/ethereum/ens.cljs]: is testnet ropsten?

**Review by 3esmit — APPROVED** [review]

**Comment by Serhy** [comment]
With https://i.diawi.com/UEFCFn (tested on iOS 13.3) and https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200206-160307-7c5dec-pr-universal.apk (on Android 8.1)
- Can attach a name from another domain
- Can buy and use stateofus.eth ENS and find by this new ENS the user
- Can see ENSs (both assigned with another domain and stateofus one) in public chats and 1-1 chat.

**Comment by flexsurfer** [comment]
@Serhy changed address for ens verification as well

**Review by cammellos — APPROVED** [review]

**Comment by Serhy** [comment]
Tested https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200207-084019-3acc45-pr.ipa on iOS 13.3 and https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200207-084018-3acc45-pr-universal.apk on Android
- Can attach a name from another domain as well as stateofus.eth
- Can buy and use stateofus.eth ENS and find by this new ENS the user
- Can see ENSs (both assigned with another domain and stateofus one) in public chats and 1-1 chat

**Pull request merged**
